src/news_gui.cpp

```diff
@@ -42,10 +42,10 @@
 
 const NewsItem *_statusbar_news_item = NULL;
 
-static uint MIN_NEWS_AMOUNT = 30;           ///< preferred minimum amount of news messages
-static uint _total_news = 0;                ///< current number of news items
-static NewsItem *_oldest_news = NULL;       ///< head of news items queue
-static NewsItem *_latest_news = NULL;       ///< tail of news items queue
+static uint MIN_NEWS_AMOUNT = 30;      ///< preferred minimum amount of news messages
+static uint _total_news = 0;           ///< current number of news items
+NewsItem *_oldest_news = NULL;         ///< head of news items queue
```

Comment:
This is visually weird. Please don't group static and non-static this close together. Also, _ means 'local'. It now is global. I believe g_ is the global identifier?

Reply:
We do not distinguish internal and external linkage in names.
We only distinguish scope, and _ prefix is for variables in global scope.